samples/languages/python/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export PATH="../../../tools:$PATH"
```

Comment:
Should we mention installing sgx-lkl (sudo -E make install) in README and using:
export PATH=/opt//sgx-lkl/bin:$PATH

Reply:
I think this should go into the installation instructions of the root README.
Created #342 for that.